playground/DetailsPage.tsx

```diff
@@ -41,6 +32,17 @@ import {
   Thumbnail,
 } from '../src';
 
+import {
```

Comment:
Is this because of icon magic imports failing?

You probably want to update these lists:

- For storybook: https://github.com/Shopify/polaris-react/blob/master/.storybook/polaris-readme-loader.js#L193
- For the styleguide examples: https://github.com/Shopify/polaris-styleguide/blob/master/client/demo-app/Demo/Demo.tsx#L11 (not strictly needed as you're not editing any README files that would then be consumed by the styleguide, but it should be kept in sync with the code in polaris-readme-loader.js